types/twig/index.d.ts

```diff
@@ -24,6 +25,7 @@ export interface Parameters {
 
 export interface Template {
     reset(blocks: any): void;
+    render(context?: any, params?: any, allow_async?: false): string;
     render(context?: any, params?: any, allow_async?: boolean): string | Promise<string>;
```

Comment:
@dmelcer9 wouldn't it be better to explicit define `false` and `true`? `false` returns `string` and `true` returns `Promise<string>` and `undefined` whatever the standard is.

Reply:
Does true always return a promise? The name of the allow_async parameter is somewhat ambiguous about this. In any case, we still need the general Boolean overload for when the parameter is unknown at compile time.

Reply:
Bumping so @typescript-bot doesn't automatically close.

Reply:
@dmelcer9 Sorry did have time to look into this and @typescript-bot seems to be very strict. Maybe we merge as it is. I looked at the code and it wasn't very clear to me, how the default is computed.